src/VsixV3/ProjectTemplates1.x/Microsoft.VisualStudio.NetCore.ProjectTemplates.1.x.vsmanproj

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I can put it there, I separated in into its own vsmanproj because its a separate package.  I didn't want ProjectTemplate.vsmanproj to fail to build because Microsoft.VisualStudio.NetCore.ProjectTemplates.1.x.vsmanproj was broken